akka-statsd-http-server/src/main/scala/statsExtension.scala

```diff
@@ -0,0 +1,19 @@
+package akka.statsd.http.server
+
+import akka.actor.{ActorRef, ExtendedActorSystem, Extension, ExtensionId}
+import akka.statsd.Stats
+import akka.statsd.{Config => StatsConfig, _}
+import java.util.concurrent.ConcurrentHashMap
```

Comment:
any reason why we want to use native java CHM instead of concurrent TrieMap? performance?

Reply:
Based upon what I read here, https://stackoverflow.com/questions/45566633/whats-the-difference-between-scala-triemap-and-java-concurrenthashmap, ConcurrentHashMap.computeIfAbsent is guaranteed to only run once, whereas TrieMap.getOrElseUpdate may run the operation multiple times if there are multiple threads calling it